.travis.yml

```diff
@@ -68,6 +68,7 @@ before_script:
 script:
     - if [ "$DISTRIB" == "ubuntu" ]; then python setup.py nosetests; fi
     - if [ "$DISTRIB" == "conda" ]; then nosetests; fi
+    - nosetests bin # to test jupyter notebook generator CLI
```

Comment:
not very conventional ... maybe it would be great if the scripts in bin, was something super simple like:

```py
from sphinx_gallery.notebook import main

if __name__ == '__main__':
    main()
```

This way you can still test the parser in sphinx_gallery/tests.